codegens/nodejs-request/lib/request.js

```diff
@@ -67,13 +70,30 @@ function makeSnippet (request, indentString, options) {
   if (options.followRedirect === false) {
     optionsArray.push(indentString + 'followRedirect: false');
   }
+  if (options.ES6_enabled === true) {
+    optionsArray.push(indentString + 'ES6_enabled: true');
```

Comment:
Why do we need this on line 74? @sastava007

Reply:
@shreys7 
I thought that it would be better to have a separate snippet for ES6 instead of using bunch of **`if`** statements. But, no problem I'll make it with single snippet only. 

And for this,  
```
 if (options.ES6_enabled === true) {
    optionsArray.push(indentString + 'ES6_enabled: true');
```
I have mistaken this and thought that we need to pass it to options property, but just I realized that it's only required till the snippet is generated and after that, it's of no use. 

**_I'll fix all this, and made a separate PR by deleting this one._**

Reply:
@sastava007 Hmm, I see. See if you can reduce the number of `if` statements. If that's causing a lot of if-else statements, then we can go with your approach.